File: Pet Stylist/Queue.java
public class Queue<T> {
    
    LinkedList<T> queue;
    
    public Queue() {
        
        queue = new LinkedList<T>();
    }
    
    public void offer(T elem) {
        queue.add(elem);
    }
    
    public T poll(){
        return queue.remove(0);
    }
    
    public T peek() {
        return queue.get(0);
    }
    
    public boolean isEmpty() {
        return queue.isEmpty();
    }
    
    public int size() {
        return queue.size();
    }
    
    
}

File: Linked Lists vs Array/queue.py
from linkedlist import LinkedList

class Queue:
    def __init__(self):
        self.queue = LinkedList()

    def offer(self, elem):
        self.queue.add(elem)

    def poll(self):
        if self.queue.isEmpty():
            return None
        return self.queue.remove(0)

    def peek(self):
        if self.queue.isEmpty():
            return None
        return self.queue.get(0)

    def isEmpty(self):
        return self.queue.isEmpty()

    def size(self):
        return self.queue.size()
